server/sandbox_run_linux.go

```diff
@@ -311,6 +311,7 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	g.SetHostname(hostname)
 
 	trusted := s.trustedSandbox(req)
+	runtimeHandler := req.GetRuntimeHandler()
```

Comment:
IMO the RunPodSandbox request should fail if the requested runtime isn't configured. This is an important part of "failing closed" so an untrusted workload can't accidentally be run unsandboxed.

Reply:
agree with that

Reply:
Yes I understand the expectation here, but this will imply the modification of a bunch of things in CRI-O as the current design always expect `func (r *Runtime) Path(c *Container) string` to return something.
@runcom @mrunalp please let me know what you think here, I don't want to introduce too many changes without agreement first.